docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-cart-rules.md

```diff
@@ -15,48 +15,241 @@ related:
+
+Use the query builder to construct queries (guided) or the **Plain query** field to enter them (free text). You can switch between both modes by clicking the corresponding button (note that incomplete queries cannot be transferred between the two modes).
+
+**Query builder**
+
+![Discount_Calculation_Query](https://spryker.s3.eu-central-1.amazonaws.com/docs/User+Guides/Back+Office+User+Guides/Discount/Discount+Calculation:+Reference+Information/query-string.png)
+
+**Plain query**
+
+![Discount_Calculation_Plain Query](https://spryker.s3.eu-central-1.amazonaws.com/docs/User+Guides/Back+Office+User+Guides/Discount/Discount+Calculation:+Reference+Information/discount-calculation-plain-query.png)
+
+The query builder lets you combine different conditions with connectors (**AND** and **OR**). Multiple conditions (rules) can be added and grouped in this way. Each condition (rule) consists of:
+* Field (for example, `attribute.color`)
+* Operator (for example, `equal(=)`)
+* Value tokens (for example, `blue`)
+
+{% info_block infoBox "Info" %}
+
+The fields and values are defined by your shop data.
+
+{% endinfo_block %}
+
+These tokens are used to build plain queries too. The pattern of the plain query is as follows:
+
+![Plain Query Pattern](https://spryker.s3.eu-central-1.amazonaws.com/docs/User+Guides/Back+Office+User+Guides/Discount/Discount+Calculation:+Reference+Information/plain-query-pattern.png)
+
+You can find plain query examples in the following table.
+
+|PLAIN QUERY|EXPLANATION|
+|---|---|
+|`day-of-week = '1'`|Discount applies if the order is placed on Monday.|
+|`shipment-carrier != '1' AND price-mode = 'GROSS_MODE'` | Discount applies if the shipment carrier with the attribute `1` is not chosen, and gross pricing is selected.|
+|`currency != 'EUR' OR price-mode = 'GROSS_MODE'` | Discount applies if the selected currency is not Euro, or the pricing mode is gross.|
+
+{% info_block infoBox "Info" %}
+
+For more information about tokens, on this page, see [Token description tables](#token-description-tables).
+
+{% endinfo_block %}
+
+**PROMOTIONAL PRODUCT**
+<br>The **PROMOTIONAL PRODUCT** discount type lets you discount specific products at a fixed quantity when the discount conditions are met. A common use case is a "buy x, get y" discount, where a customer gets an item for free when they buy a certain product or spend a certain amount.
+
+The following table describes attributes you enter if, for **DISCOUNT APPLICATION TYPE**, you select **PROMOTIONAL PRODUCT**:
+
+| ATTRIBUTE | DESCRIPTION |
+| --- | --- |
+| ABSTRACT PRODUCT SKU(S) | Stores comma-separated abstract product SKUs—products which will be promoted. |
+| MAXIMUM QUANTITY | Represents the limit on the number of units eligible for the discount. |
```

Comment:
```suggestion
| MAXIMUM QUANTITY | Maximum number of product units that can be discount. |
```

Reply:
`Eligible` is more suitable when a product is qualified for a discount, not it's number.

Reply:
Fix this and the previous one in all the reference information sections of discounts.